website/docs/docs/dbt-versions/release-notes/02-Nov-2023/job-notifications-rn.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Enhancement: Email and Slack job notifications"
+description: "November 2023: New quality-of-life improvements for setting up and administering email and Slack job notifications"
+sidebar_label: "Enhancement: Job notifications"
+sidebar_position: 10
+tags: [Nov-2023]
+---
+
+There are new quality-of-life improvements in dbt Cloud for email and Slack notifications about your jobs: 
+
+- You can add external email addresses and receive job notifications from them. External emails can be addresses that are outside of your dbt Cloud account and also for configuring notifications in third-party services like Microsoft Teams and PagerDuty. 
```

Comment:
Thoughts on this slight rephrase?

"You can add external email addresses and send job notifications to them. External emails can be addresses that are outside of your dbt Cloud account, or third-party integration addresses for configuring notifications to services like Microsoft Teams or PagerDuty.